jekyll/_cci2/configuration-reference.md

```diff
@@ -260,13 +270,28 @@ So if in the previous example `mkdir` failed to create a directory and returned
     shell: /bin/sh
     command: |
       echo Running test
-      set +e
       mkdir -p /tmp/test-results
       make test
 ```
 
-In general we recommend using the `-e` option (on by default) because it shows errors in intermediate commands and simplifies debugging in case of job failure.
+**`-o pipefail` option**
+
+> If pipefail is enabled, the pipeline’s return status is the value of the last (rightmost) command to exit with a non-zero status, or zero if all commands exit successfully. The shell waits for all commands in the pipeline to terminate before returning a value.
+
+For example:
+``` YAML
+- run: make test | tee test-output.log
+```
+
+If `make test` fails `-o pipefail` option will cause whole step to fail. Without `-o pipefail` step will always stay successful because the result of whole pipeline will be determined by the last command (`tee test-output.log`) which will always return zero status regardless of `make test`.
+
+Note that even if `make test` fails the rest of pipeline will be execute.
+
+If you want to avoid this behaviour you can specify `set +o pipefail` in command or override whole `shell` (see example above).
```

Comment:
The one at the end of "-e" section, it's logically about the same thing - how to change behavior back